<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceTimeShiftWizard</class>
 <widget class="QWizard" name="ServiceTimeShiftWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time shift wizard</string>
  </property>
  <widget class="QWizardPage" name="sourcePage">
   <property name="title">
    <string>Services time range</string>
   </property>
   <property name="subTitle">
    <string>From the main calendar view (filtered by year, church and priest) you can select more narrow time range</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Date from:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="dateFromEdit">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Date to:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="dateToEdit">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="selectionPage">
   <property name="title">
    <string>Select services to be shifted in time</string>
   </property>
   <property name="subTitle">
    <string>Shifted will be just services selected in the table, you can use also buttons to select particular weekday(s)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="serviceTable">
      <property name="toolTip">
       <string>Holy services in selected cells will be shifted in time</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="timeshiftPage">
   <property name="title">
    <string>Time shift</string>
   </property>
   <property name="subTitle">
    <string>All selected services will be shifted by given time interval - positive values move into future, negative into the past</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="yesterdayButton">
        <property name="text">
         <string>-1 day</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="oneHourEarlier">
        <property name="text">
         <string>-1 hour</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="timeShiftMinutesBox">
        <property name="toolTip">
         <string>Define time shift for selected holy services</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="suffix">
         <string> min</string>
        </property>
        <property name="minimum">
         <number>-10800</number>
        </property>
        <property name="maximum">
         <number>10800</number>
        </property>
        <property name="singleStep">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="oneHourLater">
        <property name="text">
         <string>+1 hour</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tomorrowBtn">
        <property name="text">
         <string>+1 day</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="shiftedTable"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
